identity/app/controllers/editprofile/ConsentsJourney.scala

```diff
@@ -84,6 +86,35 @@ trait ConsentsJourney
       }
     }
 
+  private def displayConsentComplete(
+    page: ConsentJourneyPage,
+    consentHint: Option[String]): Action[AnyContent] =
+    csrfAddToken {
+      consentsRedirectAction.async { implicit request =>
+        val returnUrl = returnUrlVerifier.getVerifiedReturnUrl(request).getOrElse(returnUrlVerifier.defaultReturnUrl)
+        consentCompleteView(
+          page,
+          returnUrl
+        )
+      }
+    }
+
+  private def consentCompleteView(
```

Comment:
Could you merge `consentCompleteView` into `displayConsentComplete` so that there is only one method.

Reply:
updated